content/docs/features.md

```diff
@@ -24,3 +24,17 @@ spec:
     deploy:
     ...
 ```
+
+## Cleanup plans
+
+If an optional `cleanup` plan is part of an operator, this plan will run when the operator's instance is being deleted. Once this plan is completed or failed, the instance will be deleted.
+Operator developers should take care that there aren't any triggers defined for this plan. Furthermore it should be expected that the steps of this plan could fail. E.g., an instance may get deleted because its deploy plan failed. In that case resources that the `cleanup` plan tries to remove might not exist on the cluster. 
```

Comment:
what does this mean? `Operator developers should take care that there aren't any triggers defined for this plan.` are you saying that if someone else than finalizer triggers this it won't work?

Reply:
"instance may get deleted because its deploy plan failed" - you mean KUDO instance? but the cleanup plan is run on the instance CR being removed. I feel like we need to be more clear here because I am bit lost here - sorry :(

Reply:
Oh yeah, that's quite ambiguous. I meant that a user might want to delete an operator because it's stuck in the `deploy` plan. In that case, a `cleanup` plan will run but probably fail as well, because its preconditions might not apply yet. Let me clarify this.